prusti-contracts/src/lib.rs

```diff
@@ -1,4 +1,7 @@
 #![no_std]
+#![feature(register_tool)]
```

Comment:
These, I think, are not needed anymore, right?

Reply:
that was needed for the `#[pure]` annotations.
Since those didn't seem to work across crates anyways or had any use I removed it.